functionalDB/functionalDB.markdown

```diff
@@ -1,4 +1,4 @@
-title: Designing a Database Like an Archaeologist
+title: An Archaeology-Inspired Database

```

Comment:
I think you've nailed it with that addition of the second headline, much smoother structure now